src/core/mod.rs

```diff
@@ -48,6 +48,8 @@ pub const NETWORK_P2P_PORT: u16 = 6265;
 // its current block-commit in a sortition
 pub const MINING_COMMITMENT_WINDOW: u8 = 6;
 
+pub const TX_BLOCK_LIMIT_PROPORTION_HEURISTIC: u64 = 20;
```

Comment:
Can we add a comment explaining how this constant is being used and why we came up with 20?

Reply:
Added in 973a6aad8b0896e3aa9abc6dd4aaf2ce5fc17a4f